_site/README.md

```diff
@@ -0,0 +1,33 @@
+## GitHub Pages AI Gallery Example
```

Comment:
do I understand correctly all contents under _site are auto-generated (with _site/index.html generated from index.md)? If that's case, should we not check in these files in repo, instead we auto-generate them during CI/CD?

Reply:
I think for this you are probably correct